Purchases/Purchasing/Purchases.swift

```diff
@@ -1260,25 +1260,41 @@ public extension Purchases {
 
     /**
      * Use this function to open the manage subscriptions page.
-     * If the manage subscriptions page can't be opened, the managementURL in the customerInfo will be opened.
-     * If managementURL is not available, the App Store's subscription management section will be opened.
-     *
+
      * - Parameter completion: A completion block that is called when the modal is closed.
      * If it was not successful, there will be an `Error`.
+
+     * If the manage subscriptions page can't be opened, the ``CustomerInfo/managementURL`` in
+     * the ``CustomerInfo`` will be opened. If ``CustomerInfo/managementURL`` is not available,
+     * the App Store's subscription management section will be opened.
+     *
+     * The `completion` block will be called when the modal is opened, not when it's actually closed.
+     * This is because of an undocumented change in StoreKit's behavior between iOS 15.0 and 15.2,
+     * where 15.0 would return when the modal was closed,
+     * and 15.2 returns when the modal is opened.
      */
     @available(watchOS, unavailable)
     @available(tvOS, unavailable)
+    @available(iOS 13.0, macOS 10.15, *)
```

Comment:
this was causing the docs to mark the method as deprecated

Reply:
Oooh good point.